### PR TITLE
RecentViewedViewController: Initial commit.

### PR DESCRIPTION
**Pull request by codesinc**
This commit adds the RecentViewedViewController view controller.
RecentViewedViewController is responsible of
representing list of Flickr recent viewed photos.

### CODE REVIEW

**Comment by codesinc** [comment]
@CodeBaseCamp

File: FlickrPhotos/FlickrPhotos/RecentViewedViewController.m [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  [super viewDidLoad];
+  self.splitViewController.preferredDisplayMode = UISplitViewControllerDisplayModeAllVisible;
+  self.splitViewController.delegate = self;
+  self.datasource = [[PhotosDataSource alloc] init];
+  self.tableView.dataSource = self.datasource;
+}
+
+- (void)viewDidAppear:(BOOL)animated {
+  NSArray<FlickrPhotoMetadata *> *recentViewed =
+      [FlickrPhotoMetadataSaver storedMetadata];
```
Comment (CodeBaseCamp) [inline FlickrPhotos/FlickrPhotos/RecentViewedViewController.m]: Fits line above.

**Comment by CodeBaseCamp** [comment]
Fix and squash.

File: FlickrPhotos/FlickrPhotos/RecentViewedViewController.m (hunk 1 above)
Comment (codesinc) [inline FlickrPhotos/FlickrPhotos/RecentViewedViewController.m]: f

**Comment by codesinc** [comment]
Done

**Comment by codesinc** [comment]
ping

**Pull request merged**